refinedstorage2-platform-forge/build.gradle

```diff
@@ -79,11 +79,14 @@ dependencies {
     implementation libs.apiguardian // mandatory as it's imported through platform-api
 
     compileOnly fg.deobf(forge.jei.common.api.get())
+    testCompileOnly fg.deobf(forge.jei.common.api.get())
     compileOnly fg.deobf(forge.jei.forge.api.get())
     // We test with JEI on Forge.
     runtimeOnly fg.deobf(forge.jei.forge.impl.get())
 
     compileOnly fg.deobf(forge.rei.forge.get())
+    compileOnly fg.deobf(forge.architectury.get())
+    compileOnly fg.deobf(forge.cloth.math.get())
```

Comment:
the last one is because the Point class is only in that artifact and not exported from the rei one. The other I can not remember right now,  will look into it again.